<launch>
  <arg name="file" default="$(find aero_mydemo)/config/shortinteraction.yaml"/>
  <arg name="speech_topic" default="/windows/voice"/>
  <arg name="host" default="192.168.10.55"/>

  <node name="shop_interaction" pkg="aero_mydemo"
        type="shop_interaction" output="screen">
     <param name="customer_x" value="1.0"/>
     <param name="customer_y" value="0.0"/>
     <param name="shelf_name" value="s_shelf"/>
     <remap from="/windows/voice" to="$(arg speech_topic)"/>
  </node>

  <node name="negomo_speech_input" pkg="negomo_enshu"
        type="input_speech_noentities.py" output="screen">
     <param name="file" value="$(arg file)"/>
     <remap from="/speech" to="$(arg speech_topic)"/>
  </node>

  <!-- negomo -->
  <!--node name="negomo" pkg="negomo_enshu"
        type="negotiation_model">
     <rosparam file="$(find negomo_enshu)/config/negomo.yaml"
               command="load"/>
  </node-->

  <node name="windows_alarm" pkg="linux_kinect"
        type="alarm.py" output="screen">
     <param name="ip" value="$(arg host)"/>
     <param name="alarm" value="Alarm2"/>
     <remap from="/topic" to="/negomo/vp/uistart"/>
  </node>

  <node name="yes_or_no" pkg="linux_kinect"
        type="single_conversation.py" output="screen">
     <remap from="/speech" to="$(arg speech_topic)"/>
     <remap from="/linux_kinect/single_conversation" to="/linux_kinect/yes_or_no"/>
     <param name="phrase" value="以上でよろしいでしょうか？"/>
     <param name="speakfor" value="2.5"/>
     <param name="listenfor" value="7.0"/>
     <param name="template" value="ja-JP;..\\..\\..\\..\\..\\aero-ros-pkg\\aero_salesperson2017\\config\\yes_or_no.xml"/>
  </node>

  <include file="$(find negomo)/examples/launch/plot.launch"/>
</launch>